vars/getJenkinsLabels.groovy

```diff
@@ -14,6 +14,8 @@ def call(String backend, String region=null) {
                           'aws-eu-north-1': 'aws-sct-builders-eu-north-1',
                           'aws-eu-central-1': 'aws-sct-builders-eu-central-1',
                           'aws-us-east-1' : 'aws-sct-builders-us-east-1-new',
+                          'gce-us-east1': 'gce-sct-builders-us-east1',
+                          'gce-us-west1': 'gce-sct-builders-us-west1',
                           'gce': 'gce-sct-builders',
```

Comment:
Do we need this option?

Reply:
They are needed to match the builders in special region.

In the past, we only used one gce region (us-east), and all gce builders have one label: gce-sct-builders.